uk_bin_collection/uk_bin_collection/councils/PlymouthCouncil.py

```
@@ -1,81 +1,202 @@
+                            "collectiveUPRN": {"value": str(user_uprn)},
+                            "collectiveGetJobStartDate": {"value": start_date},
+                            "collectiveGetJobEndDate": {"value": end_date},
+                        }
+                    },
+                    "isPublished": True,
+                    "formName": "Waste - Check your bin day",
+                    "processId": "AF-Process-31283f9a-3ae7-4225-af71-bf3884e0ac1b",
+                }
+
+                collective_rows = run_lookup(COLLECTIVE_JOBS_LOOKUP_ID, collective_data)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)